apps/condo/domains/miniapp/access/SendB2BAppPushMessageService.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * Generated by `createservice miniapp.SendB2BAppPushMessageService '--type=mutations'`
+ */
+const { throwAuthenticationError } = require('@open-condo/keystone/apolloErrorFormatter')
+
+const { SERVICE } = require('@condo/domains/user/constants/common')
+
+async function canSendB2BAppPushMessage (args) {
+    const { authentication: { item: user } } = args
+
+    if (!user) return throwAuthenticationError()
+    if (user.deletedAt) return false
+
+    return user.type === SERVICE
```

Comment:
Should we add some context-checks here? Or only inside mutation?

I guess we should call some service user utils here